src/server.c

```diff
@@ -4677,10 +4683,20 @@ sds genRedisInfoString(const char *section) {
     /* Persistence */
     if (allsections || defsections || !strcasecmp(section,"persistence")) {
         if (sections++) info = sdscat(info,"\r\n");
+        double fork_perc = 0;
+        if (server.stat_module_progress) {
+            fork_perc = server.stat_module_progress * 100;
+        } else if (server.stat_current_save_keys_total) {
+            fork_perc = ((double)server.stat_current_save_keys_processed / server.stat_current_save_keys_total) * 100;
+        }
+  
         info = sdscatprintf(info,
             "# Persistence\r\n"
             "loading:%d\r\n"
             "current_cow_size:%zu\r\n"
+            "current_fork_perc:%.2f\r\n"
```

Comment:
This isn't shown as a percentage (71.84 vs 71.84%), it looks like 2/3 of the "percentages" in info add the % to the end. I suppose we should also add it here.

Reply:
Good catch. 
It was actually part of my suggestion, but I guess lost in copy paste and I missed it.